prisma/schema.prisma

```diff
@@ -193,6 +193,8 @@ model PlaygroundRequest {
 
   acceptedAt DateTime?
 
+  feedbackRequested Boolean?
```

Comment:
To handle applications, accepted at different times, this flag should propably be in the playgroundapplication, alongside an acceptedAt date time field.